launch/tier4_perception_launch/launch/object_recognition/detection/detection.launch.xml

```diff
@@ -70,11 +70,9 @@
       <arg name="trust_object_distance" value="$(var trust_object_distance)"/>
       <arg name="use_low_height_cropbox" value="$(var use_low_height_cropbox)"/>
       <arg name="use_roi_based_cluster" value="$(var use_roi_based_cluster)"/>
-      <arg name="use_low_height_cropbox" value="$(var use_low_height_cropbox)"/>
-      <arg name="use_roi_based_cluster" value="$(var use_roi_based_cluster)"/>
+      <arg name="detection_by_tracker_param_path" value="$(var detection_by_tracker_param_path)"/>
```

Comment:
camera_lidar_radar based fusion seems not using this parameter.

https://github.com/autowarefoundation/autoware.universe/blob/580765f07ccdb0129fbfdf3bada3c57782c88345/launch/tier4_perception_launch/launch/object_recognition/detection/camera_lidar_radar_fusion_based_detection.launch.xml#L222

Could you fix it in this PR?

Reply:
https://github.com/autowarefoundation/autoware.universe/pull/5664
This PR will change to use the parameter, so I think the fix doesn't need.